fe/src/pages/user/Profile/Edit/ProfileEdit.jsx

```diff
@@ -20,37 +27,36 @@ function ProfileEdit() {
 
   return (
     <Fragment>
-      <Header />
-      <div className="container card p-2">
-        <div className="row gx-2">
-          <div className="col-lg-4 col-md-4 mb-2">
-            <div className="p-3 card bg-light">
-              <div className="d-flex justify-content-center">
-                <img
-                  className="mx-auto rounded-circle"
-                  style={{ width: 200, height: 200 }}
-                  src={currentUser.avatar ?? "/images/default_image.jpg"}
-                  alt="avatar"
+      <Container style={"card p-2"}>
+        <GridRow style={"gx-2"}>
+          <GridColumn style={"col-lg-4 col-md-4 mb-2"}>
+            <Card style={"p-3 bg-light"}>
```

Comment:
- maybe you try to split the page by sections for better navigation of files. for example `line:1 - 60` you move out to `./components/HeaderSection`
- or `line:61 - 139` move out and name `./components/MainSection` 
- https://github.com/framgia/sph-els-rene/blob/refactor/FE-Create-Reusable-Components/fe/src/pages/user/Profile/Edit/ProfileEdit.jsx